tests/bwc/test_recovery.py

```diff
@@ -14,7 +14,8 @@
     UpgradePath('4.5.x', '4.6.x'),
     UpgradePath('4.6.x', '4.7.x'),
     UpgradePath('4.7.x', '4.8.x'),
-    UpgradePath('4.8.x', 'latest-nightly')
+    UpgradePath('4.8.x', '5.0.x')
```

Comment:
recovery tests are [skipped](https://github.com/crate/crate-qa/blob/fdef622b8eadf60334c1435b318268b449034828/tests/bwc/test_recovery.py#L23) currently  - but I guess probably need it anyway when eventually we unskilled them